api/supabase/queries/userPlants.ts

```diff
@@ -44,3 +44,15 @@ export async function getCurrentUserPlantsByUserId(
   }
   return data;
 }
+export async function removeUserPlantById(id: UUID) {
```

Comment:
omg i just realized that we actually don't want to delete the plants from their lists, but instead include date_removed, so that it is no longer fetched